docs/book/src/topics/full-multitenancy-implementation.md

```diff
@@ -0,0 +1,259 @@
+# Multitenancy setup with EKS and Service Account
+
+See [multitenancy](./multitenancy.md) for more
+details on enabling the functionality and the various options you can use.
+
+In this example, we are going to see how to create the following architecture with cluster API:
```

Comment:
A high level summary would be nice here like:

Creating a KinD cluster as bootstrap cluster with manager credentials, then creating an EKS-managed cluster and pivoting the management cluster to it. Then creating EKS-managed workload clusters in a different account using managed credentials.